control-plane/p4RuntimeSerializer.cpp

```diff
@@ -1372,13 +1374,14 @@ getMatchFields(const IR::P4Table* table, ReferenceMap* refMap, TypeMap* typeMap)
             matchType = MatchField::TERNARY;
 
             // If this is a ternary match on the result of an isValid() call, desugar
-            // that to a ternary match against the '$valid$' field of the appropriate
+            // that to a ternary match against the '_valid' field of the appropriate
```

Comment:
`_valid` would seem problematic if a user defined a header with a `_valid` field.  It would seem better to use something that can't collide with a user name (`$valid` as users can't use `$` in names?)